Cannot compilue under Ubuntu Trusty 14.04

```
src/core/track.cpp: In lambda function:
src/core/track.cpp:243:34: error: ‘av_rescale’ was not declared in this scope
   return av_rescale(Dur, Num, Den);
                                  ^
src/core/track.cpp: In member function ‘void FFMS_Track::FillAudioGaps()’:
src/core/track.cpp:247:73: error: ‘const void ExpectedSamples’ has incomplete type
  const auto ExpectedSamples = DurationToSamples(back().PTS - front().PTS);
                                                                         ^
src/core/track.cpp:269:74: error: ‘const void ExpectedStartSample’ has incomplete type
   const auto ExpectedStartSample = DurationToSamples(Frame.PTS - FirstPTS);
                                                                          ^
make: *** [src/core/track.lo] Error 1
```

--- a/include/ffmscompat.h
+++ b/include/ffmscompat.h
@@ -88,9 +88,13 @@ static void av_frame_free(AVFrame **frame) { av_freep(frame); }
 #	if VERSION_CHECK(LIBAVUTIL_VERSION_INT, <, 51, 42, 0, 51, 74, 100)
 #		define AVPixelFormat PixelFormat
 #		define FFMS_PIX_FMT(x) PIX_FMT_##x
-#		define FFMS_PIX_FMT_FLAG(x) PIX_FMT_##x
 #	else
 #		define FFMS_PIX_FMT(x) AV_PIX_FMT_##x
+#	endif
+
+#	if VERSION_CHECK(LIBAVUTIL_VERSION_INT, <, 52, 11, 0, 52, 32, 100)
+#		define FFMS_PIX_FMT_FLAG(x) PIX_FMT_##x
+#	else
 #		define FFMS_PIX_FMT_FLAG(x) AV_PIX_FMT_FLAG_##x
 #	endif
 

--- a/src/core/track.cpp
+++ b/src/core/track.cpp
@@ -27,6 +27,7 @@
 
 #include <libavutil/avutil.h>
 #include <libavutil/common.h>
+#include <libavutil/mathematics.h>
 
 namespace {
 FrameInfo ReadFrame(ZipFile &stream, FrameInfo const& prev, const FFMS_TrackType TT) {

--- a/src/core/track.cpp
+++ b/src/core/track.cpp
@@ -25,9 +25,11 @@
 
 #include <algorithm>
 
+extern "C" {
 #include <libavutil/avutil.h>
 #include <libavutil/common.h>
 #include <libavutil/mathematics.h>
+}
 
 namespace {
 FrameInfo ReadFrame(ZipFile &stream, FrameInfo const& prev, const FFMS_TrackType TT) {

